src/types/request.ts

```diff
@@ -9,7 +9,8 @@ export interface SearchBaseCriteria {
 }
 
 export interface SearchByZoneCriteria extends SearchBaseCriteria {
-  points: MapPoint[]
+  points: MapPoint[],
+  radius: number;
```

Comment:
```suggestion
  radius?: number;
```

Se è opzionale solo per il cerchio